specification/dimensions/commitmentdiscountid.md

```diff
@@ -0,0 +1,32 @@
+# Commitment Discount ID
+
+A commitment-based discount is a contractual commitment to use a certain amount of a specific type of usage for a fixed term in exchange for a discounted unit price. Commitment-based discounts can cover either usage or cost of various resources such as virtual machines or databases. A provider can offer various pricing models, payment options, and term durations for commitment-based discounts.
```

Comment:
@kk09v @flanakin We need a single definition here (that hopefully also aligns with the FinOps Foundation definition - or that we suggest back to the foundation). CommitmentType PR has a different blurb describing this same thing.

Karl, could we prioritize Commitment-discounts in the glossary first since it's actively being worked on and referenced in many PRs.

Reply:
Actually, to reduce inter-dependencies, @flanakin could we settle on one definition and go with it and Karl can do a sweep and move things into the glossary as he's doing the other ones as well.